Gameplay/entities/crystals/Stalagmite.cpp

```diff
@@ -18,20 +18,29 @@ void Hachiko::Scripting::Stalagmite::ActiveEffects()
 	_explosion_effect->SetActive(true);
 }
 
-void Hachiko::Scripting::Stalagmite::Falling(float fall_progress)
+void Hachiko::Scripting::Stalagmite::Falling(float fall_progress, GameObject* player)
```

Comment:
As sonar also says, you can make this 
```suggestion
void Hachiko::Scripting::Stalagmite::Falling(float fall_progress, const GameObject* player)
```